db/migrate/20190907063344_create_proofing_costs.rb

```diff
@@ -0,0 +1,17 @@
+class CreateProofingCosts < ActiveRecord::Migration[5.1]
+  disable_ddl_transaction!
```

Comment:
If we don't add the index concurrently we should be able to run this migration in a transaction